_posts/2018-11-30-Rates-Rebate.md

```diff
@@ -0,0 +1,453 @@
+
+[Appendix One](#Go-to-Appendix1) contains service blueprints that describe how the current service works, the alpha used in the Tauranga pilot and a potential future state of a fully automated proactive service future state.
+
+## How the alpha works - user interface design
+### Making it easy
+<img src = "/assets/media/Rates-Rebates/RR9.png" width = "425px" align = "right">To make it easy we built a web application (web app) for the ratepayer applicants to use as the basis of the alpha. Applicants can learn if they were eligible for a Rates Rebate in the current year by entering their address. We created a widget where the applicant only types their address, which is auto-completed for them, and their Rates appear - it uses publicly available data which we obtained in collaboration with the Tauranga City Council. Once the applicant enters their income and number of dependants, they get an indication of their Rates Rebate. If they provide accurate information, they will get an accurate calculation - it’s all in one place,  the user interface design guides people through the process giving them the information needed to help them make a decision whether its worthwhile for them to apply or not.
+
+If the applicant calculated they are eligible and chooses to apply, they are then presented with more information to ensure they understand the definitions of income and dependants. We were able to eliminate the need for paper in this process.
+ 
+Once an application is submitted online, the applicant then must visit the Tauranga City Council service centre, where they then make their statutory declaration on a touchscreen iPad using a stylus. The signature is saved and can be sent digitally to DIA. We could not avoid asking the applicant to travel to a city council service centre, because the legislation requires a witnessed declaration.
+
+The web application frontend was built in ReactJS, with a jsonapi.org server created with JSONAPI Suite on Ruby on Rails. Signatures were captured using “signature_pad” npm module. The web app was built in five weeks. All components used were open source, and available on github.com for other agencies to use.
+
+### Eligibility calculator 
+Our research told us people applying for the rates rebate often don’t know how much they earn. As many are on Superannuation or Work and Income benefits we can ask if they’re on one of these income sources, as the amounts are known.
+
+We also found that there was some embarrassment about income and about asking for any form of help which was a barrier to applying, so we designed an anonymous calculation step at the beginning of the service so people don’t have to go through a full application before they find out if they are eligible for anything (which is the current process). We found users that had gone through days of effort to apply, only to discover they were entitled to 30 cents.
+
+![User journey for pilot digital application](/assets/media/Rates-Rebates/RR10.png)
```

Comment:
this tells the reader it's a user journey, but we also need to provide a description, that includes the information portrayed, in the image - so users who can't see image will still be able to access the info.